### PR TITLE
Minor client changes

### PR DESCRIPTION
**Pull request by shriyash-shukla**

### CODE REVIEW

**Comment by stjet** [comment]
Some of the UI will change once I merge one of the branches I am working on, so I will hold off on merging this PR until after that.

**Comment by shriyash-shukla** [comment]
I'm going to add some more things before resolving the conflicts.

**Comment by stjet** [comment]
Got it

**Comment by shriyash-shukla** [comment]
Fixed, I'm a bit late 😆.

**Comment by stjet** [comment]
Ok, these changes seem reasonable

File: src/routes/minting/+page.svelte
Comment (stjet) [inline src/routes/minting/+page.svelte:1]: Please leave the deleted in, maybe as "Also, consider generating a new seed just for Banano NFTs"

Comment (stjet) [reply src/routes/minting/+page.svelte:1]: Ah ok it seems I can just edit directly. But I want to say, when minting, it really is suggested that they mint from a new seed (or specifically, new address). Minting from a personal address with eg faucet and folding payments will clutter things up, making it harder to crawl and less obvious that it is a NFT minter!

**Review by stjet — APPROVED** [review]

File: src/routes/minting/+page.svelte
Comment (shriyash-shukla) [reply src/routes/minting/+page.svelte:1]: Alright, thanks.

**Pull request merged**
